content/4_Gold/Modular.mdx

```diff
@@ -367,12 +384,30 @@ int main() {
 	}
 	{
 		mi a = 1e8, b = 1e8, c = 1e8;
-		// cout << a * b * c << "\n";  // errors out- we have to convert it to an int
+		// cout << a * b * c << "\n";  // Errors- we have to cast this an an int
 		cout << (int) (a * b * c) << "\n";  // 49000000
 	}
 }
 ```
 
+And one using AtCoder's:
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+#include <atcoder/modint>
+using mint = atcoder::modint;
+
+int main() {
+	mint a = 1e8, b = 1e8, c = 1e8;
+	mint res = a * b * c;
+	cout << res.val() << endl;  // Same result
```

Comment:
this code doesn't compile

Reply:
how should i fix it

Reply:
try running it locally and see what errors you get? we should only add this code if we expect users to actually run it. 

or we can not add this if we think the module is fine as is. I suggested adding it just to make it consistent w/ the other module.